src/metabase/models/serialization/base.clj

```diff
@@ -258,6 +258,39 @@
 (defmethod extract-one :default [model-name _opts entity]
   (extract-one-basics model-name entity))
 
+(defmulti serdes-descendants
```

Comment:
moved this up from under because this is part of the serialization process.

Reply:
Fair. `serdes-descendants` was added for selective serialization late in the process, so I guess it landed low in the file.